nuxeo-drive-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-drive/nuxeo-drive-edit-button.html

```diff
@@ -65,7 +65,10 @@ <h1>[[i18n('driveEditButton.dialog.heading')]]</h1>
         }
       },
 
-      _isAvailable: function(blob) {
+      _isAvailable: function(doc, blob) {
+        if (!doc || !this.hasPermission(doc, 'Write')) {
+          return false;
+        }
```

Comment:
Couldn't this be rewritten in a shorter way?
```
return doc && this.hasPermission(doc, 'Write') && !!blob;
```

Reply:
Agree.